src/components/JUPInput/JUPInput.tsx

```diff
@@ -100,7 +104,7 @@ const JUPInput: React.FC<JUPInputProps> = ({ placeholder, fetchInputValue, fetch
         break;
 
       default:
-        customOptions = {};
+        customOptions = {} as never;
```

Comment:
I didn't come across that definition for `never`. In my searching it seemed that `{} as never` meant an object that couldn't have properties assigned to it (only a default object). and this made the 'any' warning happy with eslint. This was a small tangent of fixing some of the eslint errors that i was doing here. I welcome other fixes if they are sane, lol

Reply:
I learned what I know about `never` from here: https://www.tutorialsteacher.com/typescript/typescript-never

I must be reading into it incorrectly though because looking at the code around that switch case more your explanation makes more sense. 

I think I'm okay with it as is I just wish I could wrap my head around it a bit more.